### ISSUE TITLE
add Netlify identity widget on admin route

### ISSUE THREAD

**Issue by Sacramentix** [delucis/astro-netlify-cms#40]
Currently if you don't set disableIdentityWidgetInjection to true then it add the Netlify Identity Widget to all page which can lead to unnecessary ressource being loaded in most case.

I would be nice to have a options to only inject it on admin routes.

**Comment by delucis** [comment]
Should be feasible. Maybe we could go for something like:

```js
NetlifyCMS({
  injectIdentityWidget: 'site' | 'admin-only' | false
})
```

This would default to `'site'` and replace the current option for disabling the widget injection.

Would you be interested in making a PR?

**Comment by SeanMcP** [comment]
Is there ever an instance where you wouldn't want identity on the admin page? An alternative might be to keep the current behavior and just include it in the `admin-dashboard.astro` component.

**Comment by delucis** [comment]
The contributor who added the feature confirmed they could log in successfully so I guess it can work in certain circumstances?

https://github.com/delucis/astro-netlify-cms/pull/33#issuecomment-1295844449

**Comment by SeanMcP** [comment]
Good to know! Being able to log in makes sense, but I wonder how you can invite users without the identity script running somewhere on the site to handle the `invite_token`.

I guess if you add the option, then it is up to the user to handle.

**Comment by delucis** [comment]
Actually tested this myself and I think comments in #33 may have been misleading. Looks like: if you were already logged in before, logging in again without the identity widget works. But if you are trying to log in for the first time (e.g. with a different browser/device or incognito window), you do need the identity widget to make that work.

Given that, I think your proposal of always including on `/admin` makes sense until we hear otherwise @SeanMcP 👍

**Comment by SeanMcP** [comment]
Sounds good. It's a small change, but I would be happy to make it if that would be helpful.

**Comment by delucis** [comment]
Absolutely — a contribution would be great!

**Comment by delucis** [comment]
Thanks again for offering to make this change @SeanMcP — decided I needed it myself for a project I was working on so got it implemented and released in v0.3.5 🙌
